README.md

```diff
@@ -7,14 +7,15 @@ A dynamic sandbox for Backend.Ai kernels.
 
  * Requirements: Docker, make
 
-As we provide all docker configurations to run this code with valid GOPATH,
-you don't have to place the cloned working copy somewhere special.
+For this project, we use Go's module system - the Go's official depedency management system built into the go command.
+
+**To run:
 
 Just run `make prepare-dev` to build and create a development container based
 on Alpine Linux.  Afterwards, you can `docker start jail-dev` and `docker
 attach jail-dev` to access its shell.
 
-Inside the container, you can use `go get`, `go build`, and so on seamlessly.
+Inside the container, run `make build`. This will build our backend.ai-jail. 
```

Comment:
Since the difference from `make build` to `make inside-container` is just an addition of `go mod tidy`, I think we can just merge those two Makefile commands into one, `build`, modifying the Dockerfiles as necessary.